lib/manager/components/validation/ServicePerModeChart.js

```diff
@@ -174,8 +176,15 @@ export default class ServicePerModeChart extends Component<Props> {
               </text>
             </g>
           ))}
+          {curtailed &&
+            <text x={100 * modes.length} y={graphHeight + 43}>
+              * only first 500 days of service shown
+            </text>
+          }
         </svg>
       </div>
     )
   }
 }
+
+const maxDaysToShow = 500
```

Comment:
I find it strange to put this constant at the end of the file. Can you move it to the top? (same for the other chart)